### PR TITLE
Add method to terminate SbDut

### PR DESCRIPTION
**Pull request by azaidy**

### CODE REVIEW

**Comment by azaidy** [comment]
Simply reusing code that exists [here](https://github.com/zeroasiccorp/switchboard/blob/main/switchboard/util.py#L50-L69) for a new terminate method.

File: switchboard/sbdut.py [hunk 1]
```diff
@@ -614,10 +616,43 @@ def simulate(
...
         return p
 
+    def terminate(
+        self,
+        stop_timeout=10,
+        use_sigint=False
+    ):
+        if not self.subprocess_list:
+            raise Exception('No ongoing simulation.'
+                'Please call simulate before trying to terminate.')
```
Comment (gadfort) [inline switchboard/sbdut.py:633]: I think you can just return, raising an exception is overkill.

File: switchboard/sbdut.py [hunk 2]
```diff
@@ -614,10 +616,43 @@ def simulate(
...
                 )
 
+        # Add newly created Popen object to subprocess list
+        self.subprocess_list.append(p)
+
         # return a Popen object that one can wait() on
 
         return p
 
+    def terminate(
```
Comment (gadfort) [inline switchboard/sbdut.py:626]: you might want to add this to SbNetwork as well?

**Review by gadfort — CHANGES_REQUESTED** [review]

File: switchboard/sbdut.py (hunk 1 above)
Comment (azaidy) [reply switchboard/sbdut.py]: Done.

File: switchboard/sbdut.py (hunk 2 above)
Comment (azaidy) [reply switchboard/sbdut.py:628]: Done.

**Review by gadfort — APPROVED** [review]
LGTM

**Pull request merged**